app/Http/Controllers/UserProjectsController.php

```diff
@@ -17,20 +17,19 @@ public function show(): Response|View
             $projectable->load('projects');
         }
 
-        if ($user->organization) {
-            $projectable = $user->organization;
-            if ($projectable->isConsultant() || $projectable->isConnector()) {
-                $section = 'contracted';
-            } elseif ($projectable->isParticipant()) {
+        if (in_array($user->context, ['individual', 'organization'])) {
+            $userContext = $user->individual ?? $user->organization;
```

Comment:
I think you can write this as follows:
```suggestion
            $userContext = $user->{$user->context}
```